magento/magento2
# 11740: Sending emails from Admin in Multi-Store Environment defaults to Primary Store

### Description
Sending emails from Admin in Multi-Store Environment defaults to Primary Store

### Fixed Issues (if relevant)
1. magento/magento2#11740: Sending emails from Admin in Multi-Store Environment defaults to Primary Store

### Manual testing scenarios
1. Create two websites: ABC.com (primary) and XYZ.com each with its store/storeview accordingly.
2. Make sure You have a different Sender Name and Sender Email for XYZ.com from ABC.com (Configuration -> General -> Store Email Address -> Sales Representative).
3. Have customer place order on XYZ.com.
4. Go into admin, click on order from XYZ.com, and resend order confirmation email.

### Contribution checklist
